.env.development

```
@@ -76,3 +76,5 @@ VITE_FEATURE_MIXPANEL=true
 VITE_FEATURE_TX_HISTORY_BYE_BYE=true
 VITE_FEATURE_SWAPPER_FIAT_RAMPS=true
 VITE_FEATURE_LEDGER_READ_ONLY=true
+
+VITE_PORTALS_BASE_URL=https://dev-api.proxy.shapeshift.com/api/v1/portals
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)